rules/library.bzl

```diff
@@ -617,6 +617,7 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
             name = swift_libname,
             module_name = module_name,
             generated_header_name = generated_swift_header_name,
+            generates_header = True,
```

Comment:
For other reviewers found why this is necessary in these [release notes](https://github.com/bazelbuild/rules_swift/releases/tag/0.20.0).